examples/misc/bin/try_dart/collection_literals.dart

```diff
@@ -0,0 +1,23 @@
+// ignore_for_file: type_annotate_public_apis
+// A list literal.
+var lostNumbers = [4, 8, 15, 16, 23, 42];
+
+// A map literal.
+var nobleGases = {
+  'He': 'Helium',
+  'Ne': 'Neon',
+  'Ar': 'Argon',
+};
+
+// A set literal.
+var frogs = {
+  'Tree',
+  'Poison dart',
+  'Glass',
+};
+
+main() {
```

Comment:
does this need a `void` return type to pass analysis?

Reply:
If we want to drop the ignore directive in line 1, then yes.